tobago-core/src/main/java/org/apache/myfaces/tobago/internal/renderkit/renderer/ButtonsRenderer.java

```diff
@@ -57,6 +57,9 @@ public void encodeBegin(final FacesContext facesContext, final UIComponent compo
             ? BootstrapClass.BTN_GROUP_VERTICAL : BootstrapClass.BTN_GROUP,
         buttons.getCustomClass());
     writer.writeAttribute(HtmlAttributes.ROLE, HtmlRoleValues.GROUP.toString(), false);
+    if (buttons.isDisabled()) {
```

Comment:
The HTML DIV tag includes the global attributes but no "disabled" attribute.
See: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/div

Imho the "disabled" attribute should not be rendered.